source/docs/02-shared.xml

```diff
@@ -333,7 +333,7 @@
                         <specDesc key="colLayout"/>
                      </specList>
                   </p>
-                  <p>In order to force a system break in the musical text <gi scheme="MEI">sb</gi> can be used.</p>
+                  <p>In order to force a new system in the musical text <gi scheme="MEI">sb</gi> can be used.</p>
```

Comment:
I am wondering if this still could confuse people to think of `<sb>` as system "break"? 

Maybe: a new system beginning?

Reply:
indeed